marlowe-tutorial/doc/marlowe-data.adoc

```diff
@@ -1,56 +1,78 @@
               | If Observation Contract Contract
               | When [Case] Timeout Contract
               | Let ValueId Value Contract
 ----
 
-We saw xref:marlowe-step-by-step#marlowe-step-by-step[in the previous tutorial] what these contracts do. In the rest of this tutorial we will dig a bit deeper into the Haskell types that are used to represent the various components of the contracts, including accounts, values, observations, and actions. We will also look at types that relate to the execution of contracts, including inputs, states, the environment. 
+We saw xref:marlowe-step-by-step#marlowe-step-by-step[in the previous tutorial] what these contracts do. In the rest of this tutorial we will dig a bit deeper into the Haskell types that are used to represent the various components of the contracts, including accounts, values, observations, and actions. We will also look at types that relate to the execution of contracts, including inputs, states, the environment.
 
 === Basic components
 
-In modelling basic parts of Marlowe we use a combination of Haskell `data` types, that define _new_ types, and `type` synonyms that give a new name to an existing type.footnote:[In fact we used `newtype` declarations rather than `data` types because they are more efficiently implemented.] So, members of `PubKey` and `Party` will look like `(PubKey "alice")`, `(PubKey "bob")` and so on. 
+In modelling basic parts of Marlowe we use a combination of Haskell `data` types, that define _new_ types, and `type` synonyms that give a new name to an existing type.footnote:[In fact we used `newtype` declarations rather than `data` types because they are more efficiently implemented.] So, members of `PubKey` and `Party` will look like `(PubKey "alice")`, `(PubKey "bob")` and so on.
 [source,haskell]
 ----
 data PubKey = PubKey Text
-type Party  = PubKey   
+type Party  = PubKey
 ----
-However, Haskell allows us to present and read in these values more concisely (by declaring a custom instance of `Show` and using _overloaded strings_) so that these can be input and read as `"alice"`, `"bob"` etc. Because `Party` is a synonym for `PubKey`, the same applies to parties to contracts. 
+However, Haskell allows us to present and read in these values more concisely (by declaring a custom instance of `Show` and using _overloaded strings_) so that these can be input and read as `"alice"`, `"bob"` etc. Because `Party` is a synonym for `PubKey`, the same applies to parties to contracts.
 
 Slot numbers and amounts of money are treated in a similar way; with the same show/overload approach they will appear in contracts as numbers:
 
 [source,haskell]
 ----
 data Slot    = Slot Integer
 type Timeout = Slot
+----
+
+Marlowe Account holds amounts of multiple currencies and/or fungible and non-fungible tokens.
+Concrete amount is indexed by a `Token`, which is a pair of `CurrencySymbol` and `TokenName`.
+You can think of Account as a `Map Token Integer`, where
```

Comment:
You can think of an Account

Reply:
Yeah, articles are not my strongest suit, sorry.